ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -102,7 +102,7 @@ func ConvertToDockerCompose(taskDefinition *ecs.TaskDefinition, metadata *LocalC
 	}
 
 	data, err := yaml.Marshal(&composeV3.Config{
-		Filename: "docker-compose.local.yml",
+		Filename: "docker-compose.ecs-local.yml",
```

Comment:
This is not completely ideal since it doesn't use the constant, but https://github.com/aws/amazon-ecs-cli/pull/851 should handle this scenario